src/hooks/useCart/useCart.tsx

```diff
@@ -138,11 +143,12 @@ export const useCart = (
       setCartState("FETCHING_QUOTA");
       try {
         const quotaResponse = await getQuota(ids, authKey, endpoint);
-        if (hasNoQuota(quotaResponse)) {
+        if (hasInvalidQuota(quotaResponse)) {
+          Sentry.captureException("Negative Quota Received");
```

Comment:
Come to think of it, it'll be good to specify the quota when capturing the exception too